config.py

```diff
@@ -28,7 +29,7 @@ def validate_config(config):
     ):
         raise ConfigError("Config error: Incorrect mergin settings")
 
-    if config.driver not in ["local", "minio"]:
+    if config.driver not in ["local", "minio", "google_drive"]:
```

Comment:
I'd suggest moving to enum (of supported drivers) rather than expanding list with strings, all comparisons in code would be less error prone.